resources/multisite_drupal_standard.make

```diff
@@ -661,24 +661,26 @@ projects[rules][patch][] = https://www.drupal.org/files/issues/2019-01-25/302844
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2160
 projects[rules][patch][] = https://www.drupal.org/files/issues/file_events-826986-31_0.patch
 
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-2615
+; We use the dev version of the module to be able to run the module simpletes.
```

Comment:
The final 't' in 'simpletest' is missing.

Reply:
Changed to simpletest